chef-powershell/spec/unit/powershell_exec_spec.rb

```diff
@@ -0,0 +1,94 @@
+#
+# Author:: Stuart Preston (<stuart@chef.io>)
+# Copyright:: Copyright (c) Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "chef-powershell"
+
+describe Chef_PowerShell::ChefPowerShell::PowerShellExec, :windows_only do
+  let(:powershell_mixin) { Class.new { include Chef_PowerShell::ChefPowerShell::PowerShellExec } }
+  subject(:object) { powershell_mixin.new }
+
+  before do
+    file_path = Gem.loaded_specs["chef-powershell"].full_gem_path + "/bin/ruby_bin_folder/#{ENV["PROCESSOR_ARCHITECTURE"]}/"
```

Comment:
Isn't this already getting set in the ruby code?

Reply:
I was using this to be explicit about which version of the code to install. In testing I found that if I didn't explicitly set this that I could run into issues where the existing chef.dll installed by Infra or Workstation were being called.